src/config_libconfig.c

```diff
@@ -320,16 +259,12 @@ char *parse_config_libconfig(options_t *opt, const char *config_file, bool *shad
 	}
 
 	config_init(&cfg);
-	{
-		// dirname() could modify the original string, thus we must pass a
-		// copy
-		char *path2 = strdup(path);
-		char *parent = dirname(path2);
-
-		if (parent)
-			config_set_include_dir(&cfg, parent);
-
-		free(path2);
+	// dirname() could modify the original string, thus we pass the allocated result
+	// of realpath().
+	char *abspath = realpath(path, NULL);
```

Comment:
Curious why the change to use realpath here? This changes the existing behavior.

Reply:
I see. Thanks for explaining!

I agree the new behavior is better.